src/NuGetGallery/Services/PackageService.cs

```diff
@@ -183,6 +183,16 @@ public override PackageRegistration FindPackageRegistrationById(string packageId
             return package;
         }
 
+        public virtual Package FilterExactPackage(
```

Comment:
Since this may return null, what do you think of this name?

```suggestion
        public virtual Package FilterExactPackageOrNull(
```

Reply:
Not sure...if this were `GetExactPackage` I would agree, but because I'm using the term `Filter` here I don't think adding `OrNull` makes as much sense. As we have `FilterLatestPackage` already which doesn't have `OrNull`.

Reply:
Added some xmldocs to define the behavior of the method more clearly.